.github/workflows/main.yaml

```diff
@@ -24,7 +24,17 @@ jobs:
       # Build and test
       ########################################
       - name: "🚀 Building"
-        run: bazel build ...
+        run: |
+          set +e -x
+          build_output=$(bazel build ... 2>&1)
```

Comment:
You must actually output this, not just swallow it.

Here is a way you can do this properly:

```
MYTMPDIR="$(mktemp -d)"
trap 'rm -rf -- "$MYTMPDIR"' EXIT
r=0
logfile="$MYTMPDIR/bazel-out.log"
bazel build ... 2>&1 | tee "$logfile" >&2 || r=?
if [ "$r" != 0 ]; then
    if grep -q 'Digests do not match" "$logfile" ; then
        export CARGO_BAZEL_REPIN=true
        bazel build ...
    else
        exit $r
    fi
fi
````